examples/storm/storm-worker-controller.json

```diff
@@ -41,12 +41,7 @@
                 "hostPort": 6703,
                 "containerPort": 6703
               }
-            ],
-            "resources": {
-              "limits": {
-                "cpu": "200m"
-              }
-            }
+            ]
```

Comment:
Is this necessary? I'd rather have limits specified everywhere to give users good examples;)